docs/pages/validator-management/vc-configuration.md

```diff
@@ -4,37 +4,24 @@ The following instructions are for stakers utilizing the Lodestar validator clie
 
 [TOC]
 
-## Wallet configuration
+### Create a keystore
```

Comment:
Should we just move this into "Setup your validator" section? would also fit in right before "Import a validator keystore to Lodestar"